README.md

```diff
@@ -20,3 +20,48 @@ This is a complete reboot of some earlier ideas that began in a Backbone app. If
 * Tolu 'The Rock' Komolafe;
 * Jesmine 'Ghost Rider' Omonori; and,
 * Bart 'Kick-Ass' Riordan.
+
+
+## Releasing:
+Right now, until we get some gulp scripts in place make sure to run the following before packaging and releasing a version:
+* `npm run build`
+* `npm run build-min`
+
+## What's included: 
+
+* Action Prototype
+Based of off facebook patterns and intended to be used with flux. It works with a loose $.ajax wrapper to fire
```

Comment:
@bartriordan for all these read me comments. Just go ahead and make specific edits in a follow up commit.